ci-operator/config/openshift/file-integrity-operator/openshift-file-integrity-operator-master.yaml

```diff
@@ -61,6 +77,28 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- as: e2e-aws-upgrade
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OO_INDEX: upgrade-bundle
+    env:
+      OO_CHANNEL: release-v0.1
+      OO_INSTALL_NAMESPACE: '!create'
+      OO_PACKAGE: file-integrity-operator
+      OO_TARGET_NAMESPACES: '!install'
```

Comment:
I'm wondering if we can specify the version without hard-coding it. The docs I've read use version strings.